README.md

```diff
@@ -41,6 +41,21 @@ This React library provides UI components, simplifying the development of modern
 
 Explore [Storybook](https://azure.github.io/communication-ui-library) to try out the UI Library today!
 
+## Data Collection
+
+The software may collect information about you and your use of the software and send it to Microsoft. Microsoft may use this information to provide services and improve our products and services. You may turn off the telemetry as described below. You can learn more about data collection and use in the help documentation and Microsoft’s [privacy statement](https://go.microsoft.com/fwlink/?LinkID=824704). For more information on the data collected by the Azure SDK, please visit the [Telemetry Guidelines](https://azure.github.io/azure-sdk/general_azurecore.html#telemetry-policy) page.
```

Comment:
https://azure.github.io/azure-sdk/general_azurecore.html#telemetry-policy is for us developing SDKs, not for developers using ACS. We should be linking out to ACS telemetry policies such as https://learn.microsoft.com/azure/communication-services/concepts/privacy